.editorconfig

```diff
@@ -0,0 +1,76 @@
+# top-most EditorConfig file
+root=true
+
+# Unix-style newlines with a newline ending every file
+[*]
+indent_style=tab
+tab_width=4
+charset=utf-8
+trim_trailing_whitespace=true
+end_of_line=lf
+insert_final_newline=true
+
+[*.{c++,cc,cpp,cxx,h,h++,hh,hpp,hxx,inl,ipp,tlh,tli}]
+
+vc_generate_documentation_comments=doxygen_slash_star
```

Comment:
Is this supposed to be slash_star or slash_star_star?  the wording of this setting implies to me it should be slash_star_star but I could be wrong here.

Reply:
Slash_star
Other options are triple_slash and xml

https://devblogs.microsoft.com/cppblog/doxygen-and-xml-doc-comment-support/